.eslintrc.yml

```diff
@@ -40,3 +40,7 @@ rules:
     - warn
   prefer-spread:
     - warn
+  indent:
+    - error
+    - 4
+    - SwitchCase: 1
```

Comment:
what is the motivation for this change?

Reply:
I believe it was eslint and prettier fighting over indentation of case within switch.